### ISSUE TITLE
Add support for multi column bullets

### ISSUE THREAD

**Issue by 0x4ym4n** [sinaatalay/RenderCV#116]
This enhancement adds multi-column bullet point support to RenderCV using LaTeX's multicol package (\usepackage{multicol}). It allows for more compact and visually appealing resume layouts by organizing bullet points into columns. This feature saves space and improves readability, especially for sections with many short points.

**Comment by sinaatalay** [comment]
Thank you for the suggestion! If we decide to do this, how would we handle an arbitrary number of bullets? Templating RenderCV is currently very easy because everything is in one column.

**Comment by 0x4ym4n** [comment]
We can set a default value for the number of columns, for instance, one. If someone requires more than one column, such as three, they can manually modify this value.

**Comment by sinaatalay** [comment]
I don't think this fits into the philosophy of RenderCV. It will complicate design options and templating.

**Comment by BrickOyster** [comment]
I know this is closed but as a suggestion
You can have your default themes be one column but also have the functionality for custom themes we create to be 2 columns
in latex minipage is a very easy way to make a document 2 columns
